cql/fhir-cql/src/main/java/com/ibm/fhir/cql/engine/searchparam/ReferenceParameter.java

```diff
@@ -68,4 +69,13 @@ public ReferenceParameter setValue(String value) {
     public String getParameterValue() {
         return getValue();
     }
+
+    @Override
+    public String toString() {
+        String rtm = null;
+        if (resourceTypeModifier != null && resourceTypeModifier.getValue() != null) {
+            rtm = resourceTypeModifier.getValue();
+        }
+        return "ReferenceParameter [resourceTypeModifier=" + rtm + ", chainedProperty=" + chainedProperty + ", value=" + value + "]";
+    }
```

Comment:
There was some funky multiline logging where ReferenceParameter caused JSON content to output as Pretty... instead I used the primitive here.